CODEOWNERS

```diff
@@ -1,14 +1,14 @@
 # Default code owner for the repository
-* @astarche @mgumble
```

Comment:
@reckenro can we add you to these spots Mose is vacating? @DavidCurtiss (or anyone else) would you like to be added to any of these sections?

This can be addressed after this PR. Not like I can stop Mose leaving with open PR comments.

Reply:
Any interest @reckenro  @DavidCurtiss  @dmondrik ?

Reply:
I've enjoyed not being responsible for reviewing everything. :) But I'm willing to step in if I'm the best person for it.